misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -90,6 +91,21 @@ data class WebConfig(
   /** If true, disables automatic load shedding when degraded. */
   val concurrency_limiter_disabled: Boolean = false,
 
+  /** The algorithm to use for determining concurrency limits. */
+  val concurrency_limiter_strategy: ConcurrencyLimiterStrategy = ConcurrencyLimiterStrategy.VEGAS,
```

Comment:
so the defaults here currently map to the [existing default](https://github.com/cashapp/misk/blob/1f766aea1bb093f86441a09a4e4316b5316f4616/misk/src/main/kotlin/misk/web/interceptors/ConcurrencyLimitsInterceptor.kt#L185-L197) (vegas limit, initialized at half of the thread pool size), a few questions:

1. Do we still need that code? what is the purpose of having a set of limiter factories? it looks like we only take the first one or use the default- does that ever alternate between limiters?

2. Should we put all of these `concurrency_limiter_*` configs in a separate config object?